.cirrus.yml

```diff
@@ -53,11 +55,11 @@ cat_logs_snippet: &CAT_LOGS
 
 merge_base_script_snippet: &MERGE_BASE
   merge_base_script:
-    - if [ "$CIRRUS_PR" = "" ]; then exit 0; fi
-    - git fetch $CIRRUS_REPO_CLONE_URL $CIRRUS_BASE_BRANCH
     - git config --global user.email "ci@ci.ci"
     - git config --global user.name "ci"
```

Comment:
Those lines could be [removed](https://github.com/bitcoin/bitcoin/commit/ac1d99240af6c5d3ed5db2beea1479903d949a37) as well.

Reply:
Oh, I see. Do you want to open a PR?

Reply:
> Oh, I see. Do you want to open a PR?

I'll handle it :)

Reply:
> Do you want to open a PR?

#1201.